Assets/Scripts/Game/UserInterfaceWindows/DaggerfallInventoryWindow.cs

```diff
@@ -1210,6 +1210,16 @@ protected void EquipItem(DaggerfallUnityItem item)
                 }
                 return;
             }
+            // If more than one item selected, equip only one
+            if (item.IsAStack())
+            {
+                DaggerfallUnityItem oneItem = playerEntity.Items.SplitStack(item, 1);
```

Comment:
Maybe could have SplitStack() add the item to the internal collection before returning the new item. Then this becomes:

if (item.IsAStack())
    item = playerEntity.Items.SplitStack(item, 1);

Just a suggestion. I think the default case should be splitting inside the ItemCollection - it can always be removed if needed for some circumstance.

Reply:
That simplifies that case, but will make other cases more awkward later; I'm thinking stack splitting to only drop/pick part of a stack, for example.
Leaving that responsibility to the caller is also more flexible, it can decide what position to use to put the items back in the list, if it so wishes.

Reply:
Fair enough :)

Reply:
Actually I went ahead and tried to implement user-controlled stack splitting (using control keys), and since the inventory uses TransferItem() between lists, your suggestion matches better existing code.
I'm only transferring one item at a time, I'm not sure how to ask the user how many items to pick/drop
https://github.com/petchema/daggerfall-unity/compare/stacking-and-equipping...petchema:stack-splitting